tls/create_certs.py

```diff
@@ -28,19 +28,17 @@ def echo_check_call(cmd):
 
 def create_key_and_cert(p):
     name = p['name']
-    domain = p['domain']
+    domains = p['domains']
     unmanaged = p.get('unmanaged', False)
     if unmanaged and namespace != 'default':
-        return
+        return {}
```

Comment:
this should probably return None or something? an empty dict feels a bit odd to me.

Reply:
I did this because the only use of `create_key_and_certs` is a dict spread, and it seemed the most uniform in terms of return type, but don't have a problem changing it if you think we should.